docs/docs/manual/running.md

```diff
@@ -92,6 +92,36 @@ To exit, close all the browser tabs, and then press `control` and `C` in the ter
 
 If you see this error, you need to [install and configure a JDK package](installing#linux), including setting up `JAVA_HOME`.
 :::
+  
+#### Running unsigned apps {#running-unsigned-apps}
+
+When you first attempt to run OpenRefine, you may receive a warning that OpenRefine is unrecognized and cannot be verified. 
+
+This warning is to be expected when attempting to start newly installed applications. This warning requires users to acknowledge opening applications from unverified sources, or unsigned applications. 
+
+If you installed OpenRefine on a machine running MacOS and double-click to open the application, the message will read "OpenRefine cannot be opened because the developer cannot be verified." If this notification appears, click "Cancel."
+
+<img src="../../../graphics/MacOSWarning.png">
```

Comment:
Why not use the pure markdown syntax of `![alt text](filename.png)` instead of HTML? Also, supplying informative alt text would be helpful.